index.js

```diff
@@ -0,0 +1,23 @@
+import './style.css';
+import {
+  display, add, modify, removeItem,
+} from './functions.js';
+import { update } from './updates.js';
+
+const TodoItemsV2 = JSON.parse(localStorage.getItem('ToDoItems')) || [];
```

Comment:
Yeah! here also use camelCase, I am thinking that maybe this is a class but that is no haha!